README.md

```diff
@@ -39,7 +29,7 @@ To run the tests to verify the behavior of the test runner, do the following:
 1. Open a terminal in the project's root
 2. Run `./bin/run-tests.sh`
 
-These are [golden tests][golden] that compare the `results.json` generated by running the current state of the code against the "known good" `tests/<test-name>/expected_results.json`. All files created during the test run itself are discarded.
+These are [golden tests][golden] that compare the `results.json` generated by running the current state of the code against the "known good" `tests/<test-name>/results.json`. All files created during the test run itself are discarded.
```

Comment:
Since it contains the expected results and not the results from the solution (necessarily), then I think the old name is correct.

Reply:
Yeah, I didn't meant to modify readme.md at all...